consensus/polybft/consensus_runtime.go

```diff
@@ -314,6 +351,11 @@ func (c *consensusRuntime) OnBlockInserted(fullBlock *types.FullBlock) {
 		c.logger.Error("failed to post block in stake manager", "err", err)
 	}
 
+	// handle governance events that happened in block
+	if err := c.governanceManager.PostBlock(postBlock); err != nil {
```

Comment:
Does it make sense to parallelize these two hooks?

Reply:
It does. But then I would say, we can parallelize all PostBlock hooks in `OnBlockInserted`. I would do this in a separate PR after an detail look if it has any implications if they run in parallel.
I created a task for this: https://polygon.atlassian.net/browse/EVM-822